pkg/apis/options/server.go

```diff
@@ -24,4 +25,8 @@ type TLS struct {
 	// Cert is the TLS certificate data to use.
 	// Typically this will come from a file.
 	Cert *SecretSource
+
+	// MinVersion is the minimal TLS version that is acceptable.
+	// E.g. Set to "TLS13", "TLS1_3" or "TLS1.3" to select TLS version 1.3.
```

Comment:
Do each of these work? I think we should try to be consistent with one choice of format for the TLS version

Reply:
Currently all of these work. 

If you think having just one documented format is preferable, then I can refactor that quickly. That would also make sense to me. Which format should we pick then, the one without space and with a dot?  `"TLS1.3" `

Reply:
I tried to compare how other tools handle this, and `curl` even allows to set the max with `1.3` only for `--tls-max`. Which also would make sense in our case here. So just `1.2` and `1.3`.
https://curl.se/docs/manpage.html

However if we look at the constant names from Go, then they picked `tls.VersionTLS12` so repeating TLS and no dots or underscores.